src/oatpp/core/async/Executor.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef oatpp_async_Executor_hpp
 #define oatpp_async_Executor_hpp
 
+#include "../base/Compiler.hpp"
```

Comment:
`..` isn't allowed in the include paths. 
Please change to "oatpp/core/base/Compiler.hpp"